src/test/util.ts

```diff
@@ -0,0 +1,22 @@
+/**
+ * @license
+ * Copyright (c) 2017 The Polymer Project Authors. All rights reserved.
+ * This code may only be used under the BSD style license found at
+ * http://polymer.github.io/LICENSE.txt
+ * The complete set of authors may be found at
+ * http://polymer.github.io/AUTHORS.txt
+ * The complete set of contributors may be found at
+ * http://polymer.github.io/CONTRIBUTORS.txt
+ * Code distributed by Google as part of the polymer project is also
+ * subject to an additional IP rights grant found at
+ * http://polymer.github.io/PATENTS.txt
+ */
+
+export function getFlowingState(stream: NodeJS.ReadableStream): boolean {
+  // Cast our streams to <any> so that we can check the flowing state.
+  // _readableState is undocumented in the Node.js TypeScript definition,
+  // however it is the supported way to assert if a stream is flowing or not.
+  // See: https://nodejs.org/api/stream.html#stream_three_states
+  const privateReadableState = (<any>stream)._readableState;
+  return privateReadableState.flowing;
```

Comment:
Nice idea, but flowing state can actually be 3 values (`true`, `false`, `null`). see: https://nodejs.org/api/stream.html#stream_three_states

While we're currently only testing "hasn't started yet" and "has started and is flowing", we'll want to be able to test "has started and is stopped" (`false`) as well. I could still pull the assert into here, but given the 3 states I prefer writing out the assertion in the test itself.